cellpy/utils/easyplot.py

```diff
@@ -0,0 +1,924 @@
+# -*- coding: utf-8 -*-
+
+# Author: Amund M. Raniseth
+# Date: 01.07.2021
+
+
+import os
+from pathlib import Path
+import logging
+import cellpy
+from matplotlib import lines
+from matplotlib.artist import kwdoc
+import matplotlib.pyplot as plt
+import matplotlib as mpl
+from matplotlib.lines import Line2D
+import numpy as np
+import warnings
+
+
+class EasyPlot():
+    # Main easyplot object. Holds user input and runs all plot commands
+

```

Comment:
Docstrings are missing - but you dont have to add them now. Can do that in the next iteration.